torch_uncertainty/__init__.py

```diff
@@ -64,16 +92,19 @@ def init_args(
 
 
 def cli_main(
-    network: pl.LightningModule,
-    datamodule: pl.LightningDataModule,
+    network: pl.LightningModule | list[pl.LightningModule],
```

Comment:
I think it is better to use List[...] from typing rather than list.

Reply:
I'll also change the | for Union to avoid breaking changes for python < 3.10